2-mandatory/8-journey-planner.js

```diff
@@ -19,7 +19,15 @@
   What's better about each approach?
 */
 
-function journeyPlanner(locations, transportMode) {}
+function journeyPlanner(locations, transportMode) {
+  let availablePlaces = [];
+  for (let key in locations) {
+    if (locations[key].indexOf(transportMode) !== -1) {
+      availablePlaces.push(key);
+    }
+  }
+  return availablePlaces;
+}
```

Comment:
Yes, this is good. Again, there are different ways of doing this so it is good to compare your answers with other trainees and discuss different approaches if you have time.